### PR TITLE
Add learning objectives

### PR DESCRIPTION
**Pull request by raar1**
This adds some learning objectives for each module, in the form of high level questions that should be answered within. I've based this also on our brainstorming session this morning.

Please have a quick skim through them and if anything obvious is missing (or shouldn't be there) just let me know :)

### CODE REVIEW

File: modules/energy-budget/info.md [hunk 1]
```diff
@@ -4,4 +4,10 @@ type: info
 order: 0
 ---
 
-Learning objectives of Module 1
\ No newline at end of file
+* Why is green computing important?
```
Comment (JaroCamphuijsen) [inline modules/energy-budget/info.md:7]:
```suggestion
In this module you will learn to answer the following questions:

* Why is green computing important?
```

File: modules/optimizing-energy-use/info.md [hunk 2]
```diff
@@ -4,4 +4,9 @@ type: info
 order: 0
 ---
 
-Learning objectives of Module 2
\ No newline at end of file
+* How do I measure how much energy I am using?
```
Comment (JaroCamphuijsen) [inline modules/optimizing-energy-use/info.md:7]:
```suggestion
In this module you will learn to answer the following questions:

* How do I measure how much energy I am using?
```

File: modules/ci-cd-handson/info.md [hunk 3]
```diff
@@ -4,4 +4,4 @@ type: info
 order: 0
 ---
 
-Learning objectives of Module 1
\ No newline at end of file
+* How do I set up or adapt my continuous integration system so that tests run only when needed?
```
Comment (JaroCamphuijsen) [inline modules/ci-cd-handson/info.md:7]:
```suggestion
In this module you will learn to answer the following questions:

* How do I set up or adapt my continuous integration and deployment system to minimize energy usage?
* What are good triggers and intervals to run my various types of tests? 
```

File: modules/hpc-handson/info.md [hunk 4]
```diff
@@ -4,4 +4,4 @@ type: info
 order: 0
 ---
 
-Learning objectives of Module 1
\ No newline at end of file
+* How do I optimize the energy usage of code I run on an HPC cluster?
```
Comment (JaroCamphuijsen) [inline modules/hpc-handson/info.md:7]:
```suggestion
In this module you will learn to answer the following questions:

* How do I optimize the energy usage of code I run on an HPC cluster?
```

File: modules/software-development-handson/info.md [hunk 5]
```diff
@@ -4,4 +4,6 @@ type: info
 order: 0
 ---
 
-Learning objectives of Module 1
\ No newline at end of file
+* How do I measure the energy performance of particular areas of my code?
```
Comment (JaroCamphuijsen) [inline modules/software-development-handson/info.md:7]:
```suggestion
In this module you will learn to answer the following questions:

* How do I measure the energy performance of particular areas of my code?
```

**Review by JaroCamphuijsen — CHANGES_REQUESTED** [review]
I suggest to add a line to give some context about the questions/list. It gives a somewhat nicer experience when landing on the page.

**Review by JaroCamphuijsen — APPROVED** [review]
Awesome!

**Comment by raar1** [comment]
Thanks @JaroCamphuijsen I agree, I've added your suggested changes.

**Comment by JaroCamphuijsen** [comment]
You're terribly quick. I also just checked whether it will actually render. It does. SO approved! Oh and by the way, we really need bullet points in our css...

**Review by esclapez — APPROVED** [review]
Very nice ! I'm thinking we might reformulate some of those questions when the material is in place, but I like it for now.

**Comment by raar1** [comment]
> You're terribly quick. I also just checked whether it will actually render. It does. SO approved! Oh and by the way, we really need bullet points in our css...

Yeah I know... We have the same problem on the RSS site. NEBULA issue?

**Comment by raar1** [comment]
Thanks @esclapez ! Yes we will indeed need to reformulate these iteratively

**Pull request merged**